snapcraft/cli/remote.py

```diff
@@ -0,0 +1,256 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright 2019 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import click
+import os
+import uuid
+
+from snapcraft.internal.remote_build import (
+    Worktree,
+    LaunchpadClient,
+    Repo,
+    InfoFile,
+    errors,
+)
+from snapcraft.formatting_utils import humanize_list
+from typing import List, Tuple
+from xdg import BaseDirectory
+from . import echo
+from ._options import get_project
+
+_SUPPORTED_ARCHS = ["amd64", "arm64", "armhf", "i386", "ppc64el", "s390x"]
+
+
+@click.group()
+def remotecli():
+    """Remote build commands"""
+    pass
+
+
+@remotecli.command("remote-build")
+@click.option(
+    "--recover",
+    metavar="<build-number>",
+    type=int,
+    nargs=1,
+    required=False,
+    help="Recover interrupted remote build.",
+)
+@click.option(
+    "--status",
+    metavar="<build-number>",
+    type=int,
+    nargs=1,
+    required=False,
+    help="Display remote build status.",
+)
+@click.option(
+    "--arch",
+    metavar="<arch-list>",
+    type=str,
+    nargs=1,
+    required=False,
+    help="Set architectures to build.",
+)
+@click.option(
+    "--git", is_flag=True, required=False, help="Build a local git repository."
+)
+@click.option(
+    "--user", metavar="<username>", nargs=1, required=False, help="Launchpad username."
+)
+def remote_build(
```

Comment:
can you pop this command just like we do for "promote"?

Reply:
Done.